utils/contentful-data-clearer/main.mjs

```diff
@@ -1,19 +1,23 @@
 import contentful from 'contentful-management';
-import 'dotenv/config'
+import 'dotenv/config';
 import { deleteContentfulContent } from "./content.mjs";
 
+const envOptions = {
+  spaceId: process.env.CONTENTFUL_SPACE_ID,
+  environmentId: process.env.CONTENTFUL_ENVIRONMENT,
+};
+
 const client = contentful.createClient(
   {
     accessToken: process.env.CONTENTFUL_MASTER_KEY,
   },
-  { type: 'plain' }
+  {
+    type: 'plain',
+    defaults: envOptions
+  }
 );
 
-const envOptions = {
-  spaceId: process.env.CONTENTFUL_SPACE_ID,
-  environmentId: process.env.CONTENTFUL_ENVIRONMENT,
-};
-
+console.log(envOptions);
```

Comment:
left a log behind

Reply:
Oh hey I was looking for him, thanks for finding him!